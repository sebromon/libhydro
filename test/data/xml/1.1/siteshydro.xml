<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T12:53:10</DateHeureCreationFichier>
    <RefFichier>fichier.xml</RefFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">25</CdIntervenant>
      <CdContact>1069</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <RefHyd>
    <SitesHydro>
      <!-- SITE 0 - 1 site avec juste un code -->
      <SiteHydro>
        <CdSiteHydro>A1984310</CdSiteHydro>
      </SiteHydro>
      <!--SITE 1 -  1 site + 3 stations -->
      <SiteHydro>
        <CdSiteHydro>O1984310</CdSiteHydro>
        <LbSiteHydro>Le Touch à Toulouse [Saint-Martin-du-Touch]</LbSiteHydro>
        <LbUsuelSiteHydro>St-Martin-du-Touch</LbUsuelSiteHydro>
        <TypSiteHydro>SOURCE</TypSiteHydro>
        <CdCommune>11354</CdCommune>
        <CdCommune>11355</CdCommune>
        <CdCommune>2B021</CdCommune>
        <StationsHydro>
          <!--STATION 1.1 -->
          <StationHydro>
            <CdStationHydro>O198431001</CdStationHydro>
            <LbStationHydro>Le Touch à Toulouse [Saint-Martin-du-Touch] - station 1</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>station 1</ComplementLibelleStationHydro>
            <NiveauAffichageStationHydro>911</NiveauAffichageStationHydro>
            <PlagesUtilStationHydro>
                <PlageUtilStationHydro>
                    <DtDebPlageUtilStationHydro>2015-02-14T11:54:06</DtDebPlageUtilStationHydro>
                    <DtFinPlageUtilStationHydro>2016-09-21T06:19:31</DtFinPlageUtilStationHydro>
                    <DtActivationPlageUtilStationHydro>2017-03-17T17:38:21</DtActivationPlageUtilStationHydro>
                    <DtDesactivationPlageUtilStationHydro>2017-04-29T19:51:48</DtDesactivationPlageUtilStationHydro>
                    <ActivePlageUtilStationHydro>0</ActivePlageUtilStationHydro>
                </PlageUtilStationHydro>
                <PlageUtilStationHydro>
                    <DtDebPlageUtilStationHydro>2020-11-03T15:02:03</DtDebPlageUtilStationHydro>
                </PlageUtilStationHydro>
            </PlagesUtilStationHydro>
          </StationHydro>
          <!--STATION 1.2 -->
          <StationHydro>
            <CdStationHydro>O198431002</CdStationHydro>
            <LbStationHydro>Le Touch à Toulouse [Saint-Martin-du-Touch] - station 2</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>station 2</ComplementLibelleStationHydro>
            <NiveauAffichageStationHydro>0</NiveauAffichageStationHydro>
          </StationHydro>
          <!--STATION 1.3 -->
          <StationHydro>
            <CdStationHydro>O198431003</CdStationHydro>
            <LbStationHydro>station 3</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
          </StationHydro>
        </StationsHydro>
      </SiteHydro>
      <!-- SITE 2 - 1 site + 1 station -->
      <SiteHydro>
        <CdSiteHydro>O2000040</CdSiteHydro>
        <LbSiteHydro>La Garonne à Toulouse</LbSiteHydro>
        <LbUsuelSiteHydro>Toulouse</LbUsuelSiteHydro>
        <TypSiteHydro>REEL</TypSiteHydro>
        <StationsHydro>
          <!--STATION 2.1 -->
          <StationHydro>
            <CdStationHydro>O200004001</CdStationHydro>
            <LbStationHydro>La Garonne à Toulouse - station 1</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>échelle principale</ComplementLibelleStationHydro>
            <CoordStationHydro>
              <CoordXStationHydro>15.0</CoordXStationHydro>
              <CoordYStationHydro>16.0</CoordYStationHydro>
              <ProjCoordStationHydro>26</ProjCoordStationHydro>
            </CoordStationHydro>
            <PkStationHydro>1.0</PkStationHydro>
          </StationHydro>
        </StationsHydro>
      </SiteHydro>
      <!-- SITE 3 - 1 site + 1 station + 2 capteurs -->
      <SiteHydro>
        <CdSiteHydro>O1712510</CdSiteHydro>
        <LbSiteHydro>L'Ariège à Auterive</LbSiteHydro>
        <LbUsuelSiteHydro>AUTERIVE</LbUsuelSiteHydro>
        <TypSiteHydro>REEL</TypSiteHydro>
        <!-- <MnSiteHydro/> -->
        <CoordSiteHydro>
          <CoordXSiteHydro>618766</CoordXSiteHydro>
          <CoordYSiteHydro>1781803</CoordYSiteHydro>
          <ProjCoordSiteHydro>26</ProjCoordSiteHydro>
        </CoordSiteHydro>
        <!-- <PkAmontSiteHydro>990000</PkAmontSiteHydro> -->
        <!-- <PkAvalSiteHydro>880000</PkAvalSiteHydro> -->
        <!-- <AltiSiteHydro> -->
        <!--   <AltitudeSiteHydro>175</AltitudeSiteHydro> -->
        <!--   <SysAltimetriqueSiteHydro>3</SysAltimetriqueSiteHydro> -->
        <!-- </AltiSiteHydro> -->
        <!-- <BassinVersantSiteHydro>69</BassinVersantSiteHydro> -->
        <!-- <FuseauHoraireSiteHydro>1</FuseauHoraireSiteHydro> -->
        <!-- <StatutSiteHydro>1</StatutSiteHydro> -->
        <!-- <DonPonctSiteHydro>false</DonPonctSiteHydro> -->
        <!-- <DtPremDonSiteHydro>2007-03-01</DtPremDonSiteHydro> -->
        <!-- <PremMoisEtiageSiteHydro>09</PremMoisEtiageSiteHydro> -->
        <!-- <PremMoisAnHydSiteHydro>03</PremMoisAnHydSiteHydro> -->
        <!-- <DroitPublicationSiteHydro>10</DroitPublicationSiteHydro> -->
        <!-- <EssaiSiteHydro>false</EssaiSiteHydro> -->
        <!-- <InfluGeneSiteHydro>2</InfluGeneSiteHydro> -->
        <CdEntiteHydrographique>Y1524018</CdEntiteHydrographique>
        <!-- <LoisStatContexteSiteHydro> -->
        <!--   <LoiStatContexteSiteHydro> -->
        <!--     <TypContexteLoiStat>1</TypContexteLoiStat> -->
        <!--     <TypLoiSiteHydro>2</TypLoiSiteHydro> -->
        <!--   </LoiStatContexteSiteHydro> -->
        <!--   <LoiStatContexteSiteHydro> -->
        <!--     <TypContexteLoiStat>2</TypContexteLoiStat> -->
        <!--     <TypLoiSiteHydro>0</TypLoiSiteHydro> -->
        <!--   </LoiStatContexteSiteHydro> -->
        <!--   <LoiStatContexteSiteHydro> -->
        <!--     <TypContexteLoiStat>3</TypContexteLoiStat> -->
        <!--     <TypLoiSiteHydro>3</TypLoiSiteHydro> -->
        <!--   </LoiStatContexteSiteHydro> -->
        <!-- </LoisStatContexteSiteHydro> -->
        <!-- <ImagesSiteHydro> -->
        <!--   <ImageSiteHydro> -->
        <!--     <AdressedelImageSiteHydro>http://image1.jpeg</AdressedelImageSiteHydro> -->
        <!--     <TypIllSiteHydro>1</TypIllSiteHydro> -->
        <!--     <FormatIllSiteHydro>image/jpeg</FormatIllSiteHydro> -->
        <!--   </ImageSiteHydro> -->
        <!--   <ImageSiteHydro> -->
        <!--     <AdressedelImageSiteHydro>http://image2.bmp</AdressedelImageSiteHydro> -->
        <!--     <TypIllSiteHydro>2</TypIllSiteHydro> -->
        <!--     <FormatIllSiteHydro>image/bmp</FormatIllSiteHydro> -->
        <!--   </ImageSiteHydro> -->
        <!-- </ImagesSiteHydro> -->
        <RolesContactSiteHydro>
           <RoleContactSiteHydro>
            <CdContact schemeAgencyID="SANDRE">2</CdContact>
            <RoleContactSiteHydro>ADM</RoleContactSiteHydro>
          </RoleContactSiteHydro>
          <RoleContactSiteHydro>
            <CdContact schemeAgencyID="SANDRE">1234</CdContact>
            <RoleContactSiteHydro>REF</RoleContactSiteHydro>
            <DtDebutContactSiteHydro>2010-05-17T11:26:39</DtDebutContactSiteHydro>
            <DtFinContactSiteHydro>2038-01-19T20:55:30</DtFinContactSiteHydro>
            <DtMAJRoleContactSiteHydro>2017-11-04T09:23:31</DtMAJRoleContactSiteHydro>
          </RoleContactSiteHydro>
        </RolesContactSiteHydro>
        <CdTronconHydrographique>O0011532</CdTronconHydrographique>
        <TronconsVigilanceSiteHydro>
          <TronconVigilanceSiteHydro>
            <CdTronconVigilance>AG3</CdTronconVigilance>
          </TronconVigilanceSiteHydro>
          <TronconVigilanceSiteHydro>
            <CdTronconVigilance>AG5</CdTronconVigilance>
            <NomCTronconVigilance>Troncon Adour àvâl</NomCTronconVigilance>
          </TronconVigilanceSiteHydro>
        </TronconsVigilanceSiteHydro>
        <CdSiteHydroAncienRef>O1235401</CdSiteHydroAncienRef>
        <StationsHydro>
          <!--STATION 3.1 - avec 2 capteurs -->
          <StationHydro>
            <CdStationHydro>O171251001</CdStationHydro>
            <LbStationHydro>L'Ariège à Auterive - station de secours</LbStationHydro>
            <TypStationHydro>DEB</TypStationHydro>
            <ComplementLibelleStationHydro>Complément du libellé</ComplementLibelleStationHydro>
            <DescriptifStationHydro>Station située à Auterive</DescriptifStationHydro>
            <DtMAJStationHydro>2017-07-17T11:23:34</DtMAJStationHydro>
            <CoordStationHydro>
              <CoordXStationHydro>15.0</CoordXStationHydro>
              <CoordYStationHydro>16.0</CoordYStationHydro>
              <ProjCoordStationHydro>26</ProjCoordStationHydro>
            </CoordStationHydro>
            <PkStationHydro>153.71</PkStationHydro>
            <DtMiseServiceStationHydro>1991-10-07T14:15:16</DtMiseServiceStationHydro>
            <DtFermetureStationHydro>2012-04-21T19:58:03</DtFermetureStationHydro>
            <ASurveillerStationHydro>true</ASurveillerStationHydro>
            <NiveauAffichageStationHydro>991</NiveauAffichageStationHydro>
            <DroitPublicationStationHydro>20</DroitPublicationStationHydro>
            <EssaiStationHydro>false</EssaiStationHydro>
            <InfluLocaleStationHydro>2</InfluLocaleStationHydro>
            <ComInfluLocaleStationHydro>Libellé influence</ComInfluLocaleStationHydro>
            <ComStationHydro>commentaire1 création station hydro</ComStationHydro>
            <StationHydroAnterieure>
                <CdStationHydro>G876542134</CdStationHydro>
            </StationHydroAnterieure>
            <StationHydroFille>
                <CdStationHydro>L854795216</CdStationHydro>
            </StationHydroFille>
            <QualifsDonneesStationHydro>
              <QualifDonneesStationHydro>
                <CdRegime>1</CdRegime>
                <QualifDonStationHydro>12</QualifDonStationHydro>
                <ComQualifDonStationHydro>Commentaire qualif</ComQualifDonStationHydro>
              </QualifDonneesStationHydro>
              <QualifDonneesStationHydro>
                <CdRegime>2</CdRegime>
                <QualifDonStationHydro>16</QualifDonStationHydro>
              </QualifDonneesStationHydro>
            </QualifsDonneesStationHydro>
            <FinalitesStationHydro>
              <CdFinaliteStationHydro>1</CdFinaliteStationHydro>
              <CdFinaliteStationHydro>2</CdFinaliteStationHydro>
            </FinalitesStationHydro>
            <LoisStatContexteStationHydro>
              <LoiStatContexteStationHydro>
                <TypContexteLoiStat>1</TypContexteLoiStat>
                <TypLoiStationHydro>1</TypLoiStationHydro>
              </LoiStatContexteStationHydro>
              <LoiStatContexteStationHydro>
                <TypContexteLoiStat>3</TypContexteLoiStat>
                <TypLoiStationHydro>2</TypLoiStationHydro>
              </LoiStatContexteStationHydro>
              <LoiStatContexteStationHydro>
                <TypContexteLoiStat>2</TypContexteLoiStat>
                <TypLoiStationHydro>3</TypLoiStationHydro>
              </LoiStatContexteStationHydro>
            </LoisStatContexteStationHydro>
            <RolesContactStationHydro>
              <RoleContactStationHydro>
                <CdContact schemeAgencyID="SANDRE">2</CdContact>
                <RoleContact>ADM</RoleContact>
                <DtDebutContactStationHydro>2005-11-18T14:56:54</DtDebutContactStationHydro>
                <DtFinContactStationHydro>2007-05-04T14:12:28</DtFinContactStationHydro>
                <DtMAJRoleContactStationHydro>2012-10-04T11:35:21</DtMAJRoleContactStationHydro>
              </RoleContactStationHydro>
              <RoleContactStationHydro>
                <CdContact schemeAgencyID="SANDRE">999</CdContact>
                <RoleContact>REF</RoleContact>
              </RoleContactStationHydro>
            </RolesContactStationHydro>
            <PlagesUtilStationHydro>
              <PlageUtilStationHydro>
                <DtDebPlageUtilStationHydro>2006-04-25T16:00:00</DtDebPlageUtilStationHydro>
                <DtFinPlageUtilStationHydro>2006-04-30T17:00:00</DtFinPlageUtilStationHydro>
                <DtActivationPlageUtilStationHydro>2007-01-18T15:10:05</DtActivationPlageUtilStationHydro>
                <DtDesactivationPlageUtilStationHydro>2014-10-11T09:47:44</DtDesactivationPlageUtilStationHydro>
                <ActivePlageUtilStationHydro>true</ActivePlageUtilStationHydro>
              </PlageUtilStationHydro>
              <PlageUtilStationHydro>
                <DtDebPlageUtilStationHydro>2006-05-25T16:00:00</DtDebPlageUtilStationHydro>
                <DtFinPlageUtilStationHydro>2006-05-30T17:00:00</DtFinPlageUtilStationHydro>
                <ActivePlageUtilStationHydro>false</ActivePlageUtilStationHydro>
              </PlageUtilStationHydro>
            </PlagesUtilStationHydro>
            <ReseauxMesureStationHydro>
              <CodeSandreRdd>10</CodeSandreRdd>
              <CodeSandreRdd>1000000001</CodeSandreRdd>
            </ReseauxMesureStationHydro>
            <Capteurs>
              <!--CAPTEUR 3.1.1 -->
              <Capteur>
                <CdCapteur>O17125100102</CdCapteur>
                <LbCapteur>Radar</LbCapteur>
                <!-- <MnCapteur>Rad</MnCapteur> -->
                <!-- <TypCapteur>2</TypCapteur> -->
                <TypMesureCapteur>H</TypMesureCapteur>
              </Capteur>
              <!--CAPTEUR 3.1.2 -->
              <Capteur>
                <CdCapteur>O17125100101</CdCapteur>
                <LbCapteur>Ultrasons principal</LbCapteur>
                <!-- <MnCapteur>UP</MnCapteur> -->
                <TypCapteur>3</TypCapteur>
                <TypMesureCapteur>H</TypMesureCapteur>
                <!-- <EssaiCapteur>false</EssaiCapteur> -->
                <!-- <ComCapteur>Capteur jaune</ComCapteur> -->
                <!-- <Observateur> -->
                <!--   <CdContact schemeAgencyID="SANDRE">3</CdContact> -->
                <!-- </Observateur> -->
                <PlagesUtilCapteur>
                  <PlageUtilCapteur>
                    <DtDebPlageUtilCapteur>2009-11-03T15:19:18</DtDebPlageUtilCapteur>
                    <DtFinPlageUtilCapteur>2015-03-21T11:14:47</DtFinPlageUtilCapteur>
                    <DtActivationPlageUtilCapteur>2014-12-14T18:27:32</DtActivationPlageUtilCapteur>
                    <DtDesactivationPlageUtilCapteur>2015-10-25T19:13:04</DtDesactivationPlageUtilCapteur>
                    <ActivePlageUtilCapteur>true</ActivePlageUtilCapteur>
                  </PlageUtilCapteur>
                  <PlageUtilCapteur>
                    <DtDebPlageUtilCapteur>2016-01-15T12:14:13</DtDebPlageUtilCapteur>
                  </PlageUtilCapteur>
                </PlagesUtilCapteur>
                <CdCapteurAncienRef>O1712510</CdCapteurAncienRef>
              </Capteur>
            </Capteurs>
            <RefsAlti>
              <RefAlti>
                <DtDebutRefAlti>2006-01-01T08:00:00</DtDebutRefAlti>
                <DtFinRefAlti>2006-01-31T10:00:00</DtFinRefAlti>
                <DtActivationRefAlti>2009-12-04T11:32:04</DtActivationRefAlti>
                <DtDesactivationRefAlti>2013-07-28T08:10:57</DtDesactivationRefAlti>
                <AltiRefAlti>
                  <AltitudeRefAlti>999</AltitudeRefAlti>
                  <SysAltiRefAlti>4</SysAltiRefAlti>
                </AltiRefAlti>
                <DtMajRefAlti>2014-04-24T16:54:21</DtMajRefAlti>
              </RefAlti>
              <RefAlti>
                <DtDebutRefAlti>2007-02-01T08:00:00</DtDebutRefAlti>
                <DtFinRefAlti>2007-02-28T10:00:00</DtFinRefAlti>
                <AltiRefAlti>
                  <AltitudeRefAlti>777</AltitudeRefAlti>
                  <SysAltiRefAlti>7</SysAltiRefAlti>
                </AltiRefAlti>
              </RefAlti>
            </RefsAlti>
            <CdStationHydroAncienRef>O1712510</CdStationHydroAncienRef>
            <CdCommune>11354</CdCommune>
          </StationHydro>
        </StationsHydro>
        <!-- <LamesDEau> -->
        <!--   <LameDEau> -->
        <!--     <CdSiteMeteo>031200001</CdSiteMeteo> -->
        <!--     <PonderationLameDEau>10</PonderationLameDEau> -->
        <!--   </LameDEau> -->
        <!--   <LameDEau> -->
        <!--     <CdSiteMeteo>031200005</CdSiteMeteo> -->
        <!--     <PonderationLameDEau>20</PonderationLameDEau> -->
        <!--   </LameDEau> -->
        <!-- </LamesDEau> -->
        <CdZoneHydro>H420</CdZoneHydro>
        <PrecisionCoursDEauSiteHydro>bras principal</PrecisionCoursDEauSiteHydro>
      </SiteHydro>
    </SitesHydro>
  </RefHyd>
</hydrometrie>
