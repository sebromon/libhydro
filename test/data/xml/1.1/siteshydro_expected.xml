<?xml version="1.0" encoding="UTF-8"?>
<hydrometrie xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd">
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T12:53:10</DateHeureCreationFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">25</CdIntervenant>
      <CdContact>1069</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <RefHyd>
    <SitesHydro>
      <SiteHydro>
        <CdSiteHydro>A1984310</CdSiteHydro>
        <TypSiteHydro>REEL</TypSiteHydro>
      </SiteHydro>
      <SiteHydro>
        <CdSiteHydro>O1984310</CdSiteHydro>
        <LbSiteHydro>Le Touch à Toulouse [Saint-Martin-du-Touch]</LbSiteHydro>
        <LbUsuelSiteHydro>St-Martin-du-Touch</LbUsuelSiteHydro>
        <TypSiteHydro>SOURCE</TypSiteHydro>
        <CdCommune>11354</CdCommune>
        <CdCommune>11355</CdCommune>
        <CdCommune>2B021</CdCommune>
        <StationsHydro>
          <StationHydro>
            <CdStationHydro>O198431001</CdStationHydro>
            <LbStationHydro>Le Touch à Toulouse [Saint-Martin-du-Touch] - station 1</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>station 1</ComplementLibelleStationHydro>
            <NiveauAffichageStationHydro>911</NiveauAffichageStationHydro>
            <PlagesUtilStationHydro>
                <PlageUtilStationHydro>
                    <DtDebPlageUtilStationHydro>2015-02-14T11:54:06</DtDebPlageUtilStationHydro>
                    <DtFinPlageUtilStationHydro>2016-09-21T06:19:31</DtFinPlageUtilStationHydro>
                    <DtActivationPlageUtilStationHydro>2017-03-17T17:38:21</DtActivationPlageUtilStationHydro>
                    <DtDesactivationPlageUtilStationHydro>2017-04-29T19:51:48</DtDesactivationPlageUtilStationHydro>
                    <ActivePlageUtilStationHydro>false</ActivePlageUtilStationHydro>
                </PlageUtilStationHydro>
                <PlageUtilStationHydro>
                    <DtDebPlageUtilStationHydro>2020-11-03T15:02:03</DtDebPlageUtilStationHydro>
                </PlageUtilStationHydro>
            </PlagesUtilStationHydro>
          </StationHydro>
          <StationHydro>
            <CdStationHydro>O198431002</CdStationHydro>
            <LbStationHydro>Le Touch à Toulouse [Saint-Martin-du-Touch] - station 2</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>station 2</ComplementLibelleStationHydro>
            <NiveauAffichageStationHydro>0</NiveauAffichageStationHydro>
          </StationHydro>
          <StationHydro>
            <CdStationHydro>O198431003</CdStationHydro>
            <LbStationHydro>station 3</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <NiveauAffichageStationHydro>0</NiveauAffichageStationHydro>
          </StationHydro>
        </StationsHydro>
      </SiteHydro>
      <SiteHydro>
        <CdSiteHydro>O2000040</CdSiteHydro>
        <LbSiteHydro>La Garonne à Toulouse</LbSiteHydro>
        <LbUsuelSiteHydro>Toulouse</LbUsuelSiteHydro>
        <TypSiteHydro>REEL</TypSiteHydro>
        <StationsHydro>
          <StationHydro>
            <CdStationHydro>O200004001</CdStationHydro>
            <LbStationHydro>La Garonne à Toulouse - station 1</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>échelle principale</ComplementLibelleStationHydro>
            <CoordStationHydro>
              <CoordXStationHydro>15.0</CoordXStationHydro>
              <CoordYStationHydro>16.0</CoordYStationHydro>
              <ProjCoordStationHydro>26</ProjCoordStationHydro>
            </CoordStationHydro>
            <PkStationHydro>1.0</PkStationHydro>
            <NiveauAffichageStationHydro>0</NiveauAffichageStationHydro>
          </StationHydro>
        </StationsHydro>
      </SiteHydro>
      <SiteHydro>
        <CdSiteHydro>O1712510</CdSiteHydro>
        <LbSiteHydro>L'Ariège à Auterive</LbSiteHydro>
        <LbUsuelSiteHydro>AUTERIVE</LbUsuelSiteHydro>
        <TypSiteHydro>REEL</TypSiteHydro>
        <CoordSiteHydro>
          <CoordXSiteHydro>618766.0</CoordXSiteHydro>
          <CoordYSiteHydro>1781803.0</CoordYSiteHydro>
          <ProjCoordSiteHydro>26</ProjCoordSiteHydro>
        </CoordSiteHydro>
        <CdEntiteHydrographique>Y1524018</CdEntiteHydrographique>
        <CdTronconHydrographique>O0011532</CdTronconHydrographique>
        <TronconsVigilanceSiteHydro>
          <TronconVigilanceSiteHydro>
            <CdTronconVigilance>AG3</CdTronconVigilance>
          </TronconVigilanceSiteHydro>
          <TronconVigilanceSiteHydro>
            <CdTronconVigilance>AG5</CdTronconVigilance>
            <NomCTronconVigilance>Troncon Adour àvâl</NomCTronconVigilance>
          </TronconVigilanceSiteHydro>
        </TronconsVigilanceSiteHydro>
        <CdSiteHydroAncienRef>O1235401</CdSiteHydroAncienRef>
        <StationsHydro>
          <StationHydro>
            <CdStationHydro>O171251001</CdStationHydro>
            <LbStationHydro>L'Ariège à Auterive - station de secours</LbStationHydro>
            <TypStationHydro>LIMNI</TypStationHydro>
            <ComplementLibelleStationHydro>Complément du libellé</ComplementLibelleStationHydro>
            <DescriptifStationHydro>Station située à Auterive</DescriptifStationHydro>
            <DtMAJStationHydro>2017-07-17T11:23:34</DtMAJStationHydro>
            <CoordStationHydro>
              <CoordXStationHydro>15.0</CoordXStationHydro>
              <CoordYStationHydro>16.0</CoordYStationHydro>
              <ProjCoordStationHydro>26</ProjCoordStationHydro>
            </CoordStationHydro>
            <PkStationHydro>153.71</PkStationHydro>
            <DtMiseServiceStationHydro>1991-10-07T14:15:16</DtMiseServiceStationHydro>
            <DtFermetureStationHydro>2012-04-21T19:58:03</DtFermetureStationHydro>
            <ASurveillerStationHydro>true</ASurveillerStationHydro>
            <NiveauAffichageStationHydro>1</NiveauAffichageStationHydro>
            <ReseauxMesureStationHydro>
              <CodeSandreRdd>10</CodeSandreRdd>
              <CodeSandreRdd>1000000001</CodeSandreRdd>
            </ReseauxMesureStationHydro>
            <Capteurs>
              <Capteur>
                <CdCapteur>O17125100102</CdCapteur>
                <LbCapteur>Radar</LbCapteur>
                <TypCapteur>0</TypCapteur>
                <TypMesureCapteur>H</TypMesureCapteur>
              </Capteur>
              <Capteur>
                <CdCapteur>O17125100101</CdCapteur>
                <LbCapteur>Ultrasons principal</LbCapteur>
                <TypCapteur>3</TypCapteur>
                <TypMesureCapteur>H</TypMesureCapteur>
                <PlagesUtilCapteur>
                  <PlageUtilCapteur>
                    <DtDebPlageUtilCapteur>2009-11-03T15:19:18</DtDebPlageUtilCapteur>
                    <DtFinPlageUtilCapteur>2015-03-21T11:14:47</DtFinPlageUtilCapteur>
                    <DtActivationPlageUtilCapteur>2014-12-14T18:27:32</DtActivationPlageUtilCapteur>
                    <DtDesactivationPlageUtilCapteur>2015-10-25T19:13:04</DtDesactivationPlageUtilCapteur>
                    <ActivePlageUtilCapteur>true</ActivePlageUtilCapteur>
                  </PlageUtilCapteur>
                  <PlageUtilCapteur>
                    <DtDebPlageUtilCapteur>2016-01-15T12:14:13</DtDebPlageUtilCapteur>
                  </PlageUtilCapteur>
                </PlagesUtilCapteur>
                <CdCapteurAncienRef>O1712510</CdCapteurAncienRef>
              </Capteur>
            </Capteurs>
            <CdStationHydroAncienRef>O1712510</CdStationHydroAncienRef>
            <CdCommune>11354</CdCommune>
          </StationHydro>
        </StationsHydro>
        <CdZoneHydro>H420</CdZoneHydro>
        <PrecisionCoursDEauSiteHydro>bras principal</PrecisionCoursDEauSiteHydro>
      </SiteHydro>
    </SitesHydro>
  </RefHyd>
</hydrometrie>
